docs/3/configuration/_execfiles.yml

```
@@ -0,0 +1,22 @@
+name: <execfiles>
+description: |-
+  The `<execfiles>` tag runs a shell command and reads its output. This tag can only be defined once.
+
+  The purpose of the output depends on what field commands are defined in.
+
+  Note that commands are only run on rehash, so changes are not automatically detected.
+attributes:
+- name: motd
+  type: Text
+  required: false
+  default: None
+  description: |-
+    A file used as the Message Of The Day, ie. shown to users when they connect or use the `/MOTD` command.
+details: |-
+  The working directory of the directory use to run InspIRCd.
+example: |-
+  Downloads `https://www.example.com/motd.txt` and uses it as Message Of The Day:
```

```suggestion
  Downloads `https://www.example.com/motd.txt` and uses it as the message of the day:
```

(We don't capitalise this.)